src/guiEngine.cpp

```diff
@@ -433,7 +433,7 @@ void GUIEngine::drawOverlay(video::IVideoDriver* driver)
 
 	video::ITexture* texture = m_textures[TEX_LAYER_OVERLAY].texture;
 
-	/* If no texture, draw background of solid color */
+	/* If no texture, draw nothing */
```

Comment:
Surely this can be:
// If no texture, draw nothing
for consistency with other comments?